versions.tf

```diff
@@ -29,5 +29,5 @@ terraform {
       version = "~> 2.3"
     }
   }
-  required_version = "~> 0.13.0"
+  required_version = "~> 1.0.11"
```

Comment:
We should use required_version = ">= 1.0.0" , so all version about 1.x can be used, not specific one.

Reply:
sure thing. i will change my local branch and test before I update the PR